.github/ISSUE_TEMPLATE/3documentation.yml

```diff
@@ -0,0 +1,31 @@
+name: Documentation issue
+description: Report a possible issue in the documentation
+labels: doc
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for submitting this issue! We'll try to triage it ASAP!
+        Please do not submit bug reports or feature requests about the package itself.  
```

Comment:
This also isn't relevant, this seems to have been copied from a website specific repo. I think we'd want issues about the package itself for everything except expressjs.com

Reply:
> Also add a note that this is specifically for DOCUMENTATION issues, not issues or bugs with the software itself

@crandmck said that it was often confusing, so adding this note wouldn't hurt. Maybe we could remove the first paragraph, what do you think?

Reply:
It makes sense for the website, but if this is applied to all packages it’ll be confusing to have any wording saying not for issues about that package. It’s the reason I checked the expressjs.com issue first, so you might want to still do a website specific template and make a different one for all other packages.

Reply:
Okay, I understand, I will delete it.